com.microsoft.mrtk.graphicstools.unity/Samples/Magnifier/Magnifier/MagnifyTest.shader

```diff
@@ -0,0 +1,94 @@
+Shader "Unlit/MagnifyTest"
```

Comment:
Is this a dupe of what is in the runtime folder?

Reply:
Yes it is, It has been deleted